README.md

```diff
@@ -33,43 +39,10 @@ another, the two machines must have the same operating system and architecture.
 If they are not, you will need to re-build the native module after deploying in
 order to get the correct binaries.
 
-During installation, the module will first attempt build from source on the
-target machine. If that fails, it will attempt to download a pre-built binary
-for your system. You can disable the download attempt by setting
-`NR_NATIVE_METRICS_NO_DOWNLOAD` to `true` in your environment before
-installation.
-
-```sh
-$ export NR_NATIVE_METRICS_NO_DOWNLOAD=true
-$ npm install @newrelic/native-metrics
-```
-
-If you would like to skip the build step and only attempt to download a
-pre-build binary, set `NR_NATIVE_METRICS_NO_BUILD` to `true`.
-
-```sh
-$ export NR_NATIVE_METRICS_NO_BUILD=true
-$ npm install @newrelic/native-metrics
-```
-
-If both env vars are set, `NO_BUILD` will override `NO_DOWNLOAD`.
-
-If you are working behind a firewall and want to cache the downloads internally
-you can set the value of the download host and remote path instead of forcing a
-build:
-
-```sh
-$ export NR_NATIVE_METRICS_DOWNLOAD_HOST=http://your-internal-cache/
-$ export NR_NATIVE_METRICS_REMOTE_PATH=path/to/download/folder/
-$ npm install @newrelic/native-metrics
-```
-
-You can also specify a proxy host to route requests through using the `NR_NATIVE_METRICS_PROXY_HOST` environment variable:
+During installation, the module will first attempt to locate a prebuilt binary for the target machine within its included list of prebuilt binaries. If that fails, it will attempt a standard [node-gyp](https://www.npmjs.com/package/node-gyp#installation) build. If you do not want to use prebuilt binary, or know that it will need to be built, you can force a build:
 
 ```sh
-$ export NR_NATIVE_METRICS_NO_BUILD=true
-$ export NR_NATIVE_METRICS_PROXY_HOST=http://your-proxy-host/
-$ npm install @newrelic/native-metrics
+$ npm install @newrelic/native-metrics --build-from-source
```

Comment:
so if i run `npm run build` it will bundle and move the prebuilt to the prebuilds folder with the appropriate  arch, abi, etc.  But if i run `npm i --build-from-source` it keeps it in build/Release/native_metrics.node`. They both work, but should we do the renaming in the install and build from source use case for consistency?

Reply:
The `build.js` is really just for our CI or local development. The `npm i --build-from-source` feature is provided by `node-gyp-build`, which in turn merely utilizes the standard `node-gyp` build system. The `node-gyp-build` script is the only thing that consults the `prebuilds/` tree (provided by `prebuildify`).

All of that to say: I wouldn't worry about where a locally built binary is stored. The `require` of it is going to do the right thing.